### ISSUE TITLE
Not Getting Value of form to the console

### ISSUE THREAD

**Issue by shubhamjaisingh** [expressjs/body-parser#228]
Hi, 
I am not getting to value from the form to the app console of my app.js
code is below. please find the error.

app.js
```js
var express = require('express');
var app = express();
var port = process.env.PORT || 5000;
var bodyParser = require('body-parser');
var nav = [{
    Link: '/devices',
    Text: 'Device'
        }, {
    Link: '/about',
    Text: 'About Us'
        }, {
    Link: '/solution',
    Text: 'Solution'
        }, {
    Link: '/signup',
    Text: 'Sign Up'
        }, {
    Link: '/signin',
    Text: 'Sign In'
        }];
var deviceRouter = require('./src/routes/deviceRoutes.js')(nav);
var adminRouter = require('./src/routes/adminRoutes.js')(nav);
var authRouter = require('./src/routes/authRoutes.js')(nav);
app.use(bodyParser.json());
app.use(bodyParser.urlencoded());
app.set('views', './src/views');
app.set('view engine', 'ejs');
app.use(express.static('public'));
app.use('/devices', deviceRouter);
app.use('/admin', adminRouter);
app.use('/auth', authRouter);
app.get('/', function (req, res) {
    res.render('index', {
        title:'Nodejs -UEDeveloper',
        nav: nav
    });
});
app.listen(port, function (err) {
    console.log('Running on port' + port);
});

authRoutes.js
var express = require('express');
var authRouter = express.Router();
var mongodb = require('mongodb').MongoClient;

var router = function(){
    authRouter.post('/signup',function (req, res) {
            console.log(req.body);
        });
    authRouter.post('/signin',function (req, res) {
            console.log(req.body);
        });

    return authRouter;
};
module.exports = router; 
```

**Comment by shubhamjaisingh** [comment]
here is the console view of the program

![sketch](https://cloud.githubusercontent.com/assets/23341133/23542840/52afe218-0015-11e7-8491-b101f25728bc.png)

**Comment by dougwilson** [comment]
Hi @shubhamjaisingh in order to determine the issue, can you please include the complete steps on how you're making the request to the app?

**Comment by shubhamjaisingh** [comment]
Hello @dougwilson,
I made a form with two fields. This form have an action for '/auth/signup'.
when form is submitted then its post function should give the values to the console. For parsing I have already imported the body-parser. but it doesn't show any data.

I included the zip file of the project.
source file is in the SRC folder.
Other files like app.js or gulpfile.js is on OTHER folder
[src.zip](https://github.com/expressjs/body-parser/files/818434/src.zip)
